nixos/modules/services/x11/desktop-managers/plasma5.nix

```diff
@@ -0,0 +1,230 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  xcfg = config.services.xserver;
+  cfg = xcfg.desktopManager.plasma5;
+  xorg = pkgs.xorg;
+
+  phononBackends = {
+    gstreamer = [
+      pkgs.phonon_backend_gstreamer
+      pkgs.gst_all.gstPluginsBase
+      pkgs.gst_all.gstPluginsGood
+      pkgs.gst_all.gstPluginsUgly
+      pkgs.gst_all.gstPluginsBad
+      pkgs.gst_all.gstFfmpeg # for mp3 playback
+      pkgs.phonon_qt5_backend_gstreamer
+      pkgs.gst_all_1.gst-plugins-base
+      pkgs.gst_all_1.gst-plugins-good
+      pkgs.gst_all_1.gst-plugins-ugly
+      pkgs.gst_all_1.gst-plugins-bad
+      pkgs.gst_all_1.gst-libav # for mp3 playback
+    ];
+
+    vlc = [
+      pkgs.phonon_qt5_backend_vlc
+      pkgs.phonon_backend_vlc
+    ];
+  };
+
+  phononBackendPackages = flip concatMap cfg.phononBackends
+    (name: attrByPath [name] (throw "unknown phonon backend `${name}'") phononBackends);
+
+  kf5 = pkgs.kf55;
+
+  plasma5 = pkgs.plasma51;
+
+  kdeApps = pkgs.kdeApps_14_12;
+
+in
+
+{
+  options = {
+
+    services.xserver.desktopManager.plasma5 = {
```

Comment:
Should we really call this `plasma5` rather than `kde5` or `kde`? I understand upstream has some wacky branding going on, but as a KDE user, I wouldn't really think of searching for "plasma" when looking for the option name to enable KDE.